tests/unit/yaml/test-flow-custom-gateway-nested-config.yml

```diff
@@ -1,12 +1,7 @@
 !Flow
-with:
-  protocol: grpc
-  port: 12345
 gateway:
   uses: test-custom-gateway.yml
```

Comment:
does it have `with` inside?

Reply:
then let's remove the `uses-with` right?

Reply:
removed